src/programs/CMakeLists.txt

```diff
@@ -51,6 +51,20 @@ foreach( program ectrans-benchmark )
   endif( HAVE_GPU )
 endforeach( program )
 
+if ( HAVE_ETRANS )
+  foreach( prec sp dp )
+    if( HAVE_${prec} )
+      ecbuild_add_executable(TARGET  ectrans-lam-benchmark-${prec}
+                             SOURCES ectrans-lam-benchmark.F90
+                             LIBS
+                               fiat
+                               parkind_${prec}
+                               trans_${prec}
```

Comment:
I think something like 
`OpenMP::OpenMP_Fortran`
is needed here, as omp_lib is used in the driver

Reply:
> I think something like `OpenMP::OpenMP_Fortran` is needed here, as omp_lib is used in the driver

I wonder if that wouldn't be the reason why we had to disable the PROGRAMS via an option when compiling in a Docker image for the build of the ectrans4py Python wheel ?